src/Compilers/CSharp/Portable/CommandLine/CSharpCommandLineParser.cs

```diff
@@ -1214,7 +1216,7 @@ public new CSharpCommandLineArguments Parse(IEnumerable<string> args, string bas
             }
 
             // add additional reference paths if specified
-            if (!string.IsNullOrWhiteSpace(additionalReferenceDirectories))
+            if (!string.IsNullOrEmpty(additionalReferenceDirectories))
```

Comment:
Changing this and the check on `keyFileSetting` because whitespace-only filenames are still apparently possible